src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
For the ones that did ship, I'd preemptively put them in PublicAPI.Shipped, as that is the "safe" answer from a breakage perspective.